core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/natlang/JSGFGenerator.java

```diff
@@ -51,14 +52,17 @@
 	 *            the name of the grammar file
 	 * @param wakeup
 	 *            the wakeup command
+	 * @param singleWakeUp
+	 *            wakeup command for a single command
 	 * @param sleep
 	 *            the sleep command
 	 * @param shutdown
 	 *            the shutdown command
 	 */
-	public JSGFGenerator(String name, String wakeup, String sleep, String shutdown) {
+	public JSGFGenerator(String name, String wakeup, String singleWakeUp, String sleep, String shutdown) {
```

Comment:
rename the wakeup variable, "wakeup" and "singleWakeUp" are not distinguished enough by name